docs/FRSKY_R9.md

```diff
@@ -21,38 +21,58 @@ The Frsky R9M transmitter module and R9MX receiver are commercially available an
   </tbody>
 </table>
 
-Comment: mLRS also supports the R9MM receiver. However, flashing the R9MM with ST-Link is really tedious and requires top soldering skills or employing other tricks, as one needs to connect to four tiny solder pads. You really should consider using the R9MX receiver instead. Note: The R9MM has a different wiring diagram than the R9MX.
+Note that R9 hardware can't connect with the other mLRS boards which support 868/915 MHz. This is because it uses the SX127x LoRa chipset, which happens to be incompatible with the newer SX126x LoRa chipset when the SF6 spreading factor used by the mLRS 19 Hz mode is selected.  The older chipset also doesn't support SF5 which is required for the mLRS 31 Hz mode.
 
 ## R9M Tx Module ##
 
-The R9M module is unfortunately limited with respect to serial ports. It provides acess to only one serial port, which moreover has inverted TTL signals. In order to use this serial port, these conditions apply:
+The R9M transmitter module is somewhat limited with respect to serial ports. It provides access to only one serial port, which moreover has inverted TTL signals. In order to use this serial port, these conditions apply:
 
-1. One needs to use a "Frsky inverter" dongle. You either can buy one or build it yourself.
+1. The serial port of the R9M module can be configured to work either as "serial" or "CLI". This is done by setting the dip switch 1 (left dip switch): off (switch down) = CLI, on (switch up) = serial. Note that the dip switch position is read only at power up, i.e., one needs to re-power the module to make any change effective.
 
-2. The serial port of the R9M module can be configured to work either as "serial" or "CLI". This is done by setting the dip switch 1 (left dip switch): off (switch down) = CLI, on (switch up) = serial. Note that the dip switch position is read only at power up, i.e., one needs to repower the module to make any change effective.
+2. One needs to somehow (un)invert the serial signals for most uses. You can buy or build a "Frsky inverter" dongle to connect a standard serial adapter.
+
+3. If you use the mLRS Lua configuration script for configuration and thus don't need the CLI for configuration, you can avoid the inverter dongle when you use one of the supported ESP32 boards with the mlrs-wifi-bridge sketch to connect via MAVLink wirelessly to a Ground Control Station.  The ESP32 can directly use the inverted serial signals.
+
+4. If you have the older 2018, "ACCST" version of the R9M, you will want to perform the Inverter Mod to allow reliable communication with the radio at higher bit rates.  This is nicely documented in the [ExpressLRS documentation](https://www.expresslrs.org/hardware/inverter-mod/).  Note that the newer 2019, "ACCESS" version of the R9M does not need this modification.
```

Comment:
Should this just be a 'Note' as opposed to part of the numbered list?  Numbered list seems to refer specifically to the single serial port.

Reply:
Yes, it should.  When I first wrote it, I mis-remembered the purpose of the resistor, thinking i was for the serial port, then, when I double checked and corrected it, I neglected to move it elsewhere.  I'll try to find a better place for it.

Reply:
the place doesn't look bad to me, just could be a Note as JLP suggests :)